composer.json

```diff
@@ -51,10 +51,10 @@
         "utopia-php/cache": "0.8.*",
         "utopia-php/cli": "0.13.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.28.*",
         "utopia-php/locale": "0.4.*",
         "utopia-php/registry": "0.5.*",
         "utopia-php/preloader": "0.2.*",
+        "utopia-php/database": "dev-add-exception-class as 0.28.0",
```

Comment:
TODO : to revert/update composer after https://github.com/utopia-php/database/pull/218 is merged